packages/terra-alert/translations/fi-FI.json

```diff
@@ -1,9 +0,0 @@
-{
```

Comment:
We don't support finish so didn't request new translations for these types. Should I keep the existing translations even with the missing keys?

Reply:
I vote to remove them if we don't support the language.